src/style.js

```diff
@@ -335,3 +335,20 @@ export function resetStyles(element, properties) {
     setStyle(element, properties[i], null);
   }
 }
+
+/**
+ * Propagates the object-fit/position element attributes as styles on the
+ * childElement.
+ * @param {!Element} element ie: amp-img
+ * @param {!Element} childElement ie: the img within amp-img
```

Comment:
nit: maybe don't call it "child", since that's not technically important.  It's just the element that gets the css put on it.

Reply:
Done